mix.exs

```diff
@@ -405,6 +405,19 @@ defmodule EMQXUmbrella.MixProject do
     ])
   end
 
+  defp enterprise_umbrella_apps(:platform) do
```

Comment:
Since we disclose the existence of fdb apps in many places, it's probably better to keep all the platform tweaks in the upstream to avoid accumulating conflicts.